src/components/TrackCard.jsx

```diff
@@ -39,13 +59,18 @@ const TrackCard = (props) => {
           />
         </div>
         <div className="space-y-1 text-sm mt-2.5 transition-all">
-          <div className="flex items-center gap-3">
-            <h3
+          <div className="flex items-center gap-3 justify-center sm:justify-start">
+          <h3
               className={`font-medium leading-none ${
                 props.trackPlaying == props.id && props.playingStatus
                   ? "text-green-400 underline"
                   : ""
-              } transition-all`}
+                } transition-all text-center sm:text-left`}
+                style={{
+                  // Apply marginLeft: 30px; on screens smaller than sm 
+                  marginLeft:  isSmallScreen  ? '30px' : '0px',
+                  transition: 'margin-left 0.2s ease', // Apply transition to margin-left
+                }}
```

Comment:
Interesting approach. I'd still like to use the default tailwind breakpoints to avoid all the extra code.